src/i18n/messages.ar.xlf

```diff
@@ -52,7 +52,7 @@
         </context-group>
         <target xml:lang="ar" state="needs-translation">Are you sure you want to <x id="ICU" equiv-text="{data.changeType, select, accept {...} reject {...} unlink {...} delete {...}}"/>
       the <x id="ICU_1" equiv-text="{data.amount, select, single {...} many {...}}"/>
-  
+
```

Comment:
not sure if we want to update xlf file manually or let it be handled by crowdin or xi18n

Reply:
Long term I agree we do not want to edit this manually, but in the short term I modified this to help @mappuji get started

Reply:
ok then @mappuji will have to update other xlf file to match it as he is building multiple language